README.md

```diff
@@ -1,49 +1,105 @@
-<p align="center">
-  <a href="https://https://vuestorefront.io/">
-  <picture>
-    <img src="https://user-images.githubusercontent.com/1626923/137092657-fb398d20-b592-4661-a1f9-4135db0b61d5.png" width="400px" height="auto"/>
-  </picture>
-  </a>
-  <h1 align="center">Vue Storefront for Vue (Nuxt 3) Boilerplate</h1>
-</p>
-
-[![GitHub Repo stars](https://img.shields.io/github/stars/vuestorefront/vue-storefront?style=social)](https://github.com/vuestorefront/vue-storefront)
-[![Twitter Follow](https://img.shields.io/twitter/follow/vuestorefront?style=social)](https://twitter.com/vuestorefront)
-[![YouTube Channel Subscribers](https://img.shields.io/youtube/channel/subscribers/UCkm1F3Cglty3CE1QwKQUhhg?style=social)](https://www.youtube.com/c/VueStorefront)
+<div align="center">
+  <a href="https://https://vuestorefront.io/"><img src="https://user-images.githubusercontent.com/1626923/137092657-fb398d20-b592-4661-a1f9-4135db0b61d5.png" alt="Vue Storefront" height="80px" /></a>
+
+  <a href="https://www.plentymarkets.com/"><img src="./plentysystems.jpg" alt="plentysystems logo" height="160px" /></a>
+
+  <h1 align="center">Vue Storefront 3 Integration With plentysystems</h1>
+</div>
+
+[![GitHub Repo stars](https://img.shields.io/github/stars/plentymarkets/storefront-nuxt3-boilerplate?style=social)](https://github.com/plentymarkets/storefront-nuxt3-boilerplate)
+[![Twitter Follow](https://img.shields.io/twitter/follow/plentymarkets?style=social)](https://twitter.com/plentymarkets)
+[![YouTube Channel Subscribers](https://img.shields.io/youtube/channel/subscribers/UCauJsvmhbPNp6ii7tCGwxMg?style=social)](https://www.youtube.com/@plentymarkets)
 [![Discord](https://img.shields.io/discord/770285988244750366?label=join%20discord&logo=Discord&logoColor=white)](https://discord.vuestorefront.io)
 
-## Getting Started with Vue Storefront
+Welcome to the plentysystems integration for Vue Storefront 3. For a technical overview of the repository structure, refer to the [Guide](GUIDE.md). The rest of the README details how to configure your plenty system and get started with the project.
+
+## plenty system setup
+
+To run this project, you need a plenty system. Get a free 30-day trial [on our website](https://www.plentymarkets.com/) or choose one of the [available editions](https://www.plentymarkets.com/product/pricing/).
```

Comment:
Wouldn't we name it plentymarkets instead of plenty?

Reply:
No, we no longer talk about plentymarkets systems. I've checked with Franz and Dennis and we're going with plentysystems systems now, or just systems. I prefer the former in this case because we're not in a system context throughout.